test/system/src/test/resources/pravega_withAuth.properties

```diff
@@ -18,4 +18,5 @@ controller.auth.enabled=true
 controller.auth.userPasswordFile=/opt/pravega/conf/passwd
 controller.auth.tokenSigningKey=secret
 autoScale.authEnabled=true
-autoScale.tokenSigningKey=secret
\ No newline at end of file
+autoScale.tokenSigningKey=secret
+hdfs.replaceDataNodesOnFailure=false
```

Comment:
Since 'hdfs.replaceDataNodesOnFailure=false' this is not a security related property it would be good to add this after 'log.level=DEBUG' as in case of the other file, so its easier to compare contents of both the files.

Reply:
fixed.